File: seongwon/Programmers/Level_0/Lv0_인덱스_바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int num1, int num2) {
    string answer = "";
    char tmp;
    tmp = my_string[num1];
    my_string[num1] = my_string[num2];
    my_string[num2] = tmp;
    answer = my_string;
    return answer;
}

File: chaerim/Programmers/Level_0/Lv0_인덱스_바꾸기.py
def solution(my_string, num1, num2):
    my_string_list = list(my_string)

    my_string_list[num1], my_string_list[num2] = my_string_list[num2], my_string_list[num1]

    answer = ''.join(my_string_list)

    return answer